### PR TITLE
Fix the Logs are not collected when the installation fails

### PR DESCRIPTION
**Pull request by zsl1549**
1、To solve the problem ： Logs are not collected when the installation fails
2、solution ：putInstallClusterLog
3、test results :Setup failed, log collection

### CODE REVIEW

File: src/components/Cluster/ShowUpdateClusterDetail/index.js [hunk 1]
```diff
@@ -34,6 +38,17 @@ class UpdateClusterDetail extends PureComponent {
       callback: data => {
         if (data) {
           const { complete, steps } = cloud.showUpdateClusterSteps(data.events);
+          if (complete && steps && steps.length > 0) {
+            globalUtil.putInstallClusterLog(enterprise, rainbondInfo, {
+              eid,
+              taskID: task.taskID,
+              status: steps[steps.length - 1].Status,
+              message: steps[steps.length - 1].Message,
+              install_step: 'createK8s',
```
Comment (barnettZQG) [inline src/components/Cluster/ShowUpdateClusterDetail/index.js:47]: install_step: updateRKE

File: src/pages/AddCluster/KClusterList/index.js [hunk 2]
```diff
@@ -125,7 +125,7 @@ export default class EnterpriseClusters extends PureComponent {
                 taskID: task.taskID,
                 status: infos[0].state,
                 message: infos[0].message,
-                install_step: provider,
+                install_step: 'createK8s',
```
Comment (barnettZQG) [inline src/pages/AddCluster/KClusterList/index.js:128]: What's the log phase here?

File: src/components/Cluster/ShowUpdateClusterDetail/index.js (hunk 1 above)
Comment (zsl1549) [reply src/components/Cluster/ShowUpdateClusterDetail/index.js:47]: Have to repair

File: src/pages/AddCluster/KClusterList/index.js (hunk 2 above)
Comment (zsl1549) [reply src/pages/AddCluster/KClusterList/index.js:128]: The first phase starts installing the cluster

**Pull request closed without merging**